File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.py
car = input()
num = [1,1,1,1]
for i in range(len(car)):
    if car[i] == 'd': num[i] = 10
    else: num[i] = 26
for i in range(len(car)-1):
    if car[i] == car[i+1]: num[i+1]-=1
print(num[0]*num[1]*num[2]*num[3])

File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.cc
#include <iostream>
#include <algorithm>
#include <string>
using namespace std;

int main() {
    string car;
    cin >> car;
    int num[4] = {1,1,1,1};
    for(int i=0;i<car.size();i++)
    {
        if(car[i] == 'd') num[i] = 10;
        else num[i] = 26;
    }
    for(int i=0;i<car.size()-1;i++)
    {
        if(car[i] == car[i+1]) num[i+1]--;
    }
    cout << num[0]*num[1]*num[2]*num[3];
}